include/boost/serialization/serialization.hpp

```diff
@@ -13,6 +13,10 @@
 #include <boost/config.hpp>
 #include <boost/serialization/strong_typedef.hpp>
 
+#if !defined(BOOST_NO_CXX11_HDR_TYPE_TRAITS)
```

Comment:
You don't need <type_traits>, versions of enable_if and is_same are in boost.core.